ansible/module_utils/dualtor_utils.py

```diff
@@ -27,13 +27,18 @@ def generate_mux_cable_facts(topology):
     vlan_address_v6, netmask_v6 = vlan_prefix_v6.split("/")
     vlan_address_v4 = ipaddress.ip_address(vlan_address_v4.decode())
     vlan_address_v6 = ipaddress.ip_address(vlan_address_v6.decode())
-    for index, intf in enumerate(enabled_interfaces):
-        is_active_active = intf in host_interfaces_active_active
+    index = 1
+    for intf in enabled_interfaces:
         mux_cable_facts[intf] = dict(
-            server_ipv4=str(vlan_address_v4 + index * 2 + 1) + "/" + netmask_v4,
-            server_ipv6=str(vlan_address_v6 + index * 2 + 1) + "/" + netmask_v6,
-            cable_type="active-active" if is_active_active else "active-standby"
+            server_ipv4=str(vlan_address_v4 + index) + "/" + netmask_v4,
+            server_ipv6=str(vlan_address_v6 + index) + "/" + netmask_v6,
```

Comment:
I think server IP should be base + index * 2 and soc IP should be base + index * 2 + 1?

Reply:
This is to be compatible with exiting dualtor topology, the current code has many places to calculate the server IPs for dualtor, so I have to ensure the mixed topology calculation won't break those.

Reply:
Got ur point, updated.